Urania/Urania.Desktop/Urania - Backup.Desktop.csproj

```diff
@@ -0,0 +1,28 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Backup? Używajcie gita ludzie pls

Reply:
Sam się zrobił, nawet nie zauważyłam :3 I nie całkiem rozumiem jak się ma do używania gita, ale domyślam się że wywalę go i będzie po kłopocie

Reply:
Sam się zrobił? Jakaś ambitna funkcja VS studio? Albo... Jak rozwiązałaś konflikty się zrobil?

Reply:
Zdaje się że pobrałam coś przez sourcetree, potem wyłączyłam VS, i z ostatnim tchnieniem powiedział że coś zostało zmienione zdalnie, ale on sobie zapamięta to co było w jego oknach przy wyłączaniu xD Ale nie zauważyłam że zapamiętał gdzieś w projekcie

Reply:
Chyba wiem o co chodzi. No to zwracam honor. Już myślałem, że sama zrobiłaś backup zamiast używać gita co cofania zmian itp. 😏